src/App.tsx

```diff
@@ -16,34 +17,92 @@ export const goodsFromServer = [
 ];
 
 export const App: React.FC = () => {
+  enum Sort {
+    None = '',
+    Alphabetic = 'alphabetic',
+    Length = 'length',
+  }
+
+  const [sortMethod, setSortmethod] = useState<Sort>(Sort.None);
+  const [isReversed, changeReverse] = useState(false);
```

Comment:
```suggestion
  const [isReversed, setIsReversed] = useState(false);
```
always try to name state as [state, setState]